modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -21,13 +21,40 @@ object SeqexecEngine {
 
   sealed trait SeqexecEvent
   object SeqexecEvent {
-    case class SequenceStart(view: SequenceView) extends SeqexecEvent
-    case class StepExecuted(view: SequenceView) extends SeqexecEvent
-    case class SequenceCompleted(view: SequenceView) extends SeqexecEvent
-    case class StepBreakpointChanged(view: SequenceView) extends SeqexecEvent
-    case class StepSkipMarkChanged(view: SequenceView) extends SeqexecEvent
-    case class SequencePauseRequested(view: SequenceView) extends SeqexecEvent
-    case class NewLogMessage(msg: LogMsg) extends SeqexecEvent
+
+    import edu.gemini.seqexec.engine
+    import edu.gemini.seqexec.engine.Event
+
+    case class SequenceStart(view: List[SequenceView]) extends SeqexecEvent
```

Comment:
Perhaps we should put `List[SequenceView]` on its own type, we seems to be using it a lot. WDYT